src/SHCore.Desktop/SHCore+SHELL_UI_COMPONENT.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Identifies the type of UI component that is needed in the shell
+    /// </summary>
+    public enum SHELL_UI_COMPONENT
```

Comment:
Thinking over it all, I guess here I might have done a mistake. I didn't put SHELL_UI_COMPONENT under a class, so now it keeps itself hanging on the PInvoke main namespace. Should I change it so it gets under ShellScalingApi or something like that?